packages/frontend-core/src/fetch/ViewFetch.ts

```diff
@@ -1,7 +1,14 @@
-import { Table, View } from "@budibase/types"
+import { Table } from "@budibase/types"
 import DataFetch from "./DataFetch"
 
-type ViewV1 = View & { name: string }
+type ViewV1 = {
```

Comment:
Should this live in the BB types package, is it a UI only thing?

Reply:
It's a UI only thing. It is the interface that the svelte files will use to use it. I will rename it to match the equivalent usages in the other fetchers